### PR TITLE
Add pmswift configuration for object

### PR DESCRIPTION
**Pull request by Christop1964**
Add installation and configuration of pmswift for object
Signed-off-by: Christoph Keil <chkeil@de.ibm.com>

### CODE REVIEW

File: roles/scale_object/node/tasks/yum/install.yml [hunk 1]
```diff
@@ -14,3 +14,19 @@
...
    disable_gpg_check: "{{ disable_gpgcheck }}"
   when: ansible_fqdn in scale_obj_nodes_list
+
+- name: install | Get installed spectrum-scale-object
+  shell: rpm -qa | grep spectrum-scale-object
+  register: scale_package_status
+  when: ansible_fqdn in scale_obj_nodes_list
+  ignore_errors: true
+  args:
+    warn: false
```
Comment (rajan-mis) [inline roles/scale_object/node/tasks/yum/install.yml:24]: Can you please add when condition to run this only rhel8 node

File: roles/scale_object/node/tasks/yum/install.yml [hunk 2]
```diff
@@ -14,3 +14,19 @@
...
+
+- name: install | Check if spectrum-scale-object package is installed
+  assert:
+    that:
+    - scale_package_status.rc == 0
+    fail_msg: "spectrum-scale-object is not installed on {{ ansible_hostname }}"
+  when: ansible_fqdn in scale_obj_nodes_list
```
Comment (rajan-mis) [inline roles/scale_object/node/tasks/yum/install.yml:31]: Can you please add when condition to run this only rhel8 node

File: roles/scale_object/node/tasks/yum/install.yml (hunk 1 above)
Comment (rajan-mis) [reply roles/scale_object/node/tasks/yum/install.yml:24]: `when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '8'`

File: roles/scale_object/node/tasks/yum/install.yml (hunk 2 above)
Comment (rajan-mis) [reply roles/scale_object/node/tasks/yum/install.yml:31]: `when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '8'`

Comment (Christop1964) [reply roles/scale_object/node/tasks/yum/install.yml:31]: Hi Rajan, as we discuss it in Slack. 
we have this check already at the include_task when we include the yum/install.yml file.
So it makes no sense to have it inside the yml as will.

- import_tasks: yum/install.yml
  when: ansible_distribution in scale_rhel_distribution

**Review by rajan-mis — APPROVED** [review]

**Pull request merged**